### PR TITLE
Fix for tests

### PR DESCRIPTION
**Pull request by andresailer**
Trying to get the tests to pass, and also make them use the local stuff, and not what is in the stack

### CODE REVIEW

**Comment by andresailer** [comment]
The one test with the converters currently fails because the release is too old and doesn't include the latest k4LCIOReader

**Comment by fdplacido** [comment]
The last commit highlights that 2 tests are failing, which fixes this issue: https://github.com/key4hep/k4MarlinWrapper/issues/46

File: test/scripts/all_events_bounds.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 fi
 
-if [ ! -f $k4MarlinWrapper_tests_DIR/inputFiles/muons.slcio ]; then
-  wget https://github.com/AIDASoft/DD4hep/raw/master/DDTest/inputFiles/muons.slcio -P $k4MarlinWrapper_tests_DIR/inputFiles/
+if [ ! -f $TEST_DIR/inputFiles/muons.slcio ]; then
+  wget https://github.com/AIDASoft/DD4hep/raw/master/DDTest/inputFiles/muons.slcio -P $TEST_DIR/inputFiles/
 fi
 
-../run k4run $k4MarlinWrapper_tests_DIR/gaudi_opts/test_k4MarlinWrapper1.py --num-events=-1
+../run k4run $TEST_DIR/gaudi_opts/simple_processors.py --num-events=-1
```
Comment (andresailer) [inline test/scripts/all_events_bounds.sh:12]:
```suggestion
k4run $TEST_DIR/gaudi_opts/simple_processors.py --num-events=-1
```

File: test/scripts/over_total_events.sh [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 fi
 
-if [ ! -f $k4MarlinWrapper_tests_DIR/inputFiles/muons.slcio ]; then
-  wget https://github.com/AIDASoft/DD4hep/raw/master/DDTest/inputFiles/muons.slcio -P $k4MarlinWrapper_tests_DIR/inputFiles/
+if [ ! -f $TEST_DIR/inputFiles/muons.slcio ]; then
+  wget https://github.com/AIDASoft/DD4hep/raw/master/DDTest/inputFiles/muons.slcio -P $TEST_DIR/inputFiles/
 fi
 
-../run k4run $k4MarlinWrapper_tests_DIR/gaudi_opts/test_k4MarlinWrapper1.py --num-events=15
+../run k4run $TEST_DIR/gaudi_opts/simple_processors.py --num-events=15
```
Comment (andresailer) [inline test/scripts/over_total_events.sh:12]:
```suggestion
k4run $TEST_DIR/gaudi_opts/simple_processors.py --num-events=15
```

File: test/scripts/simple_processors.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+if [ ! -d $TEST_DIR/inputFiles/ ]; then
+  mkdir $TEST_DIR/inputFiles
+fi
+
+if [ ! -f $TEST_DIR/inputFiles/muons.slcio ]; then
+  wget https://github.com/AIDASoft/DD4hep/raw/master/DDTest/inputFiles/muons.slcio -P $TEST_DIR/inputFiles/
+fi
+
+../run k4run $TEST_DIR/gaudi_opts/simple_processors.py
```
Comment (andresailer) [inline test/scripts/simple_processors.sh:12]:
```suggestion
k4run $TEST_DIR/gaudi_opts/simple_processors.py
```

**Comment by vvolkl** [comment]
The doctest should be fixed in the latest (2021-11-26, yet to be announced) release.

**Pull request merged**